src/tests/EdFi.DataManagementService.Tests.E2E/Features/Resources/UpdateResourcesValidation.feature

```diff
@@ -56,158 +56,168 @@ Feature: Resources "Update" Operation validations
                     "shortDescription": "Sick Leave"
                   }
                   """
-        @ignore
+
         Scenario: Verify updating a non existing resource with valid data
              # The id value should be replaced with a non existing resource
-             When a PUT request is made to "ed-fi/absenceEventCategoryDescriptors/{id}" with
+             When a PUT request is made to "ed-fi/absenceEventCategoryDescriptors/00000000-0000-4000-a000-000000000000" with
                   """
-                    {
-                        "id": {id},
-                        "codeValue": "Sick Leave",
-                        "description": "Sick Leave Edited",
-                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
-                        "shortDescription": "Sick Leave"
-                    }
+                  {
+                    "id": "00000000-0000-4000-a000-000000000000",
+                    "codeValue": "Sick Leave",
+                    "description": "Sick Leave Edited",
+                    "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                    "shortDescription": "Sick Leave"
+                  }
                   """
              Then it should respond with 404
               And the response body is
                   """
-                    {
-                        "detail": "Resource to update was not found.",
-                        "type": "urn:ed-fi:api:not-found",
-                        "title": "Not Found",
-                        "status": 404,
-                        "correlationId": null
-                    }
+                  {
+                      "detail": "Resource to update was not found",
+                      "type": "urn:ed-fi:api:not-found",
+                      "title": "Not Found",
+                      "status": 404,
+                      "correlationId": null,
+                      "validationErrors": null,
+                      "errors": null
+                  }
                   """
-        @ignore
-        Scenario: Verify error handling updating a resource with invalid data
+        Scenario: Verify error handling updating a resource natural key
```

Comment:
The term to use is "updating the document identity"

Reply:
Renamed.